_content/excerpts/can-monks-practice-astrology_kotyk-j.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Can Monks Practice Astrology?: Astrology and the Vinaya in China"
+authors:
+  -  "Jeffrey Kotyk"
+external_url: "https://www.academia.edu/34408763/_Can_Monks_Practice_Astrology_Astrology_and_the_Vinaya_in_China_"
+source url: https://www.buddhismuskunde.uni-hamburg.de/pdf/4-publikationen/hamburg-buddhist-studies/hamburgup-hbs09-full.pdf
+drive_links:
+  - "https://drive.google.com/file/d/1FEAJAhNQwGIgvO1dYEY1csVcPJBLxnJh/view?usp=drive_link"
+course: medieval
+tags:
+  - astrology
+  - east-asian-roots
+  - vinaya-studies
+year: 2017
+booktitle: "Rules of Engagement: Medieval Traditions of Buddhist Monastic Regulation"
+publisher: "Hamburg University Press"
+address: "Hamburg, Germany"
+chapter: 17
+pages: "503-–517"
```

Comment:
Needs to be two hyphens. Somehow you typed a hyphen and a dash 😖 This took me a while to find!